examples/resize/resize.go

```diff
@@ -0,0 +1,66 @@
+// goncurses - ncurses library for Go.
+
+/* This example demonstrates the ability to resize. Only one of detecting SIGWINCH or KEY_RESIZE
+ * is strictly needed, but depending on the options ncurses was built with, one or the other may
+ * work better. */
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	gc "github.com/rthornton128/goncurses"
+)
+
+var stdscr *gc.Window
+
+func main() {
+	start := time.Now()
```

Comment:
I'm not sure the relevance to tracking the runtime of this code in the context of an example. Should it be removed since it is more of a distraction and not relevant to the example?

Reply:
I was using it to make sure I wasn't blocking while I was trying to build the example, but I'll remove it now.